docs/conf/na/2017/news/schedule-social.rst

```diff
@@ -0,0 +1,61 @@
+:template: 2017/na-content.html
+
+.. post:: May 1, 2017
+   :tags: 2017, portland, schedule, social
+
+Announcing Full Schedule and Social Events
+==========================================
+
+Our Portland conference is just two weeks away and we hope you're getting excited!
+
+We are officially sold-out but you can sign up for the `waiting list <https://ti.to/writethedocs/write-the-docs-na-2017>`_ in case we have any last-minute cancellations.
+
+Full Presentation Schedule
+--------------------------
+
+Our website is now updated with the `final schedule <http://www.writethedocs.org/conf/na/2017/schedule/>`_ for the presentations, so you can start planning your time at the conference.
+
+Soon we will also publish the timeslots for the unconference sessions as well as a preliminary list of Writing Day projects that you can hack on.
+
+Of course you can still sign up for unconference sessions and lightning talks during the conference, or bring your own Writing Day project.
+
+Workshops Almost Full
+---------------------
+
+The response to the new workshops has been great and we're nearly at full occupancy for both workshops. If you're still on the fence, don't wait to secure your spot.
+
+You can still grab the last tickets for the workshops on our `ticket website <https://ti.to/writethedocs/write-the-docs-na-2017>`_.
+
+Social Events
+-------------
+
+This year our main social event will be held again at the `Jackknife <https://goo.gl/maps/hvYkv6RU4qD2>`_ at 614 SW 11th Ave, on **Monday evening at 7pm**. Drinks and light snacks will be provided.
+
+We're also happy to invite you to an after-conference hangout sponsored by our friends at New Relic. The hangout will be at `New Relic <https://www.meetup.com/Write-The-Docs-PDX/events/239146623/>`_ at 6:30pm.Drinks and light snacks will be provided.
+
+And lastly, there are still a few spots left for the hike, you can register for it `here <https://ti.to/writethedocs/write-the-docs-na-2017>`_.
```

Comment:
The hike is now full at 50 people -- we'll probably need to break it up into 2, but still trying to figure out logistics -- but we should probably mention it selling out.